### ISSUE TITLE
Bad request sent to server during wallet creation

### ISSUE THREAD

**Issue by adidasboy517** [dappnode/DAppNodePackage-prysm-pyrmont#10]
On version 0.1.2.  When I upload my keystore for wallet generation and enter the same wallet password as in the Config > Advanced Editor > Validator, I encounter the following error message "Bad request sent to server, something went wrong".  Goerli Geth is synced and I've tried two different keystores (remembering the correct passwords).  I've cleared the password requirements on each step (8+ char, at least 1 uppercase and 1 lowercase, at least 1 number, and at least 1 special character like !).

**Comment by eduadiez** [comment]
Could you could add a screenshot of the process and some logs. Has it been an update? `WALLET_PASSWORD` should be empty and in this version it shouldn't be used. This is my advanced config:

<img width="1698" alt="Screenshot 2020-11-24 at 22 48 13" src="https://user-images.githubusercontent.com/20141918/100155231-2f206380-2ea7-11eb-81ae-adbfab7f5208.png">

**Comment by adidasboy517** [comment]
![Screen Shot 2020-11-24 at 4 03 30 PM](https://user-images.githubusercontent.com/60804080/100158306-93283500-2e71-11eb-89ea-eaf4998cb099.png)
![Screen Shot 2020-11-24 at 4 04 03 PM](https://user-images.githubusercontent.com/60804080/100158308-94596200-2e71-11eb-9f8a-0608b1696cf8.png)
![Screen Shot 2020-11-24 at 4 04 18 PM](https://user-images.githubusercontent.com/60804080/100158310-94f1f880-2e71-11eb-9281-3e748e66aa2e.png)
![Screen Shot 2020-11-24 at 4 04 39 PM](https://user-images.githubusercontent.com/60804080/100158311-94f1f880-2e71-11eb-98a9-cf2e1f58ed40.png)
[logs-dappnode-package-DAppNodePackage-beacon-chain.prysm-pyrmont.dnp.dappnode.eth.txt](https://github.com/dappnode/DAppNodePackage-prysm-pyrmont/files/5593086/logs-dappnode-package-DAppNodePackage-beacon-chain.prysm-pyrmont.dnp.dappnode.eth.txt)
[logs-dappnode-package-DAppNodePackage-validator.prysm-pyrmont.dnp.dappnode.eth.txt](https://github.com/dappnode/DAppNodePackage-prysm-pyrmont/files/5593087/logs-dappnode-package-DAppNodePackage-validator.prysm-pyrmont.dnp.dappnode.eth.txt)

I tried restarting my NUC, so I think that's why the beacon chain log is showing some issues now, but I was still getting the error even before when the beacon chain log looked good.

**Comment by adidasboy517** [comment]
here is my browser console error log.  I even made a third keystore (carefully typing and retyping and remembering the keystore pw) and uploaded the deposit file to the launchpad and sent 32 goeth to the contract: [https://goerli.etherscan.io/tx/0x849d0fa9ca69a06e26840900711b06a026fabed9172007a84973ed577e4041c5]
![Screen Shot 2020-11-24 at 6 41 37 PM](https://user-images.githubusercontent.com/60804080/100167626-18691500-2e85-11eb-874b-3741e1e78e98.png)
[logs-dappnode-package-DAppNodePackage-beacon-chain.prysm-pyrmont.dnp.dappnode.eth.txt](https://github.com/dappnode/DAppNodePackage-prysm-pyrmont/files/5593645/logs-dappnode-package-DAppNodePackage-beacon-chain.prysm-pyrmont.dnp.dappnode.eth.txt)
Beacon chain running fine.

**Comment by 0xtakamaka** [comment]
same issue here, using 0.1.3. See details below for further explanation.

Before filing a new issue, please provide the following information.
Core DAppNode Packages versions

    dappmanager.dnp.dappnode.eth: 0.2.35, commit: cbd7cbcf
    core.dnp.dappnode.eth: 0.2.39
    vpn.dnp.dappnode.eth: 0.2.7, commit: 12ae223f
    wifi.dnp.dappnode.eth: 0.2.4
    ipfs.dnp.dappnode.eth: 0.2.11
    bind.dnp.dappnode.eth: 0.2.6

System info

    docker version: Docker version 18.09.8-ce, build 0dd43dd87fd530113bf44c9bba9ad8b20ce4637f
    docker compose version: docker-compose version 1.25.5, build unknown
    platform: linux, x64, 5.9.0-2-amd64
    Disk usage: 29%

![slika](https://user-images.githubusercontent.com/12158250/100233477-160cc680-2f2a-11eb-9175-01a2ef1da187.png)
![slika](https://user-images.githubusercontent.com/12158250/100233193-ae567b80-2f29-11eb-9ff4-5ad35b957ef6.png)
![slika](https://user-images.githubusercontent.com/12158250/100233212-b6162000-2f29-11eb-835f-a7abd5350282.png)
![slika](https://user-images.githubusercontent.com/12158250/100233325-e1007400-2f29-11eb-8229-157c55cf4286.png)

**Comment by sinasab** [comment]
+1 experiencing the same issue as OP. Interestingly it seems that the failed request is sending the password from the prior form instead of the current one... but that password should also be passing the validation fine, as it also requires numbers/letters/special chars.

Is there a workaround to using your wallet with the prysm validator without going through the web UI in the meantime?

**Comment by adidasboy517** [comment]
You're correct, it's the webui password (middle pw step between the keystore pw and the wallet password) and it seems to need to be sufficiently complex.  I've found that a min of 7 letters (at least 1 uppercase), min of 3 numbers, and 1 special character works.  Going down to 6 letters or 2 numbers (sometimes) doesn't seem to work.  Hope this helps.  I suggest that the min requirements displayed for the WebUI password be updated to reflect this prior to closing this issue.

**Comment by sinasab** [comment]
I also just discovered this after perusing through the codepath that produced the error.

If you're getting this error, try using a more complicated/long password!

I agree that it'd be useful for the UI to surface the suggestion to use a stronger password. It's confusing otherwise, since the UI only has certain requirements, but the server has stricter requirements.

**Comment by 0xtakamaka** [comment]
And also, do not upload entire "validator_keys" folder into the browser, upload just proper .json file. 

It would be awesome if browser would just pop-up console.log messages :).

**Comment by muitosabao** [comment]
I think I'm having the same issue. How can I do this again? Can I only install the  package again and wait for the beacon chain to download again? Or is there a simpler way?

**Comment by AVaunt-consulting** [comment]
you should be able to go straight to http://prysm-pyrmont.dappnode/onboarding
to restart the process

**Comment by julienphilbert** [comment]
Hello,

Same issue

Any update on this?

Thanks

**Comment by 0xtakamaka** [comment]
> Hello,
> 
> Same issue
> 
> Any update on this?
> 
> Thanks

Check 2 things:
- your password should be complex enough
- do not upload entire validator_keys folder, just keys .json
